resources/lib/simple_logging.py

```diff
@@ -21,26 +22,32 @@ def __str__(self):
         return "LogLevel: " + str(self.level)
 
     def error(self, msg):
-        if(self.level >= 0):
+        if (self.level >= 0):
             try:
                 xbmc.log(self.format(msg, "ERROR"), level=xbmc.LOGNOTICE)
             except UnicodeEncodeError:
                 xbmc.log(self.format(msg, "ERROR").encode('utf-8'), level=xbmc.LOGNOTICE)
 
+    def warning(self, msg):
```

Comment:
I dont think 4 levels of logging are needed, it is either info or error i.e. you would like to know about it if logging is turned on or you NEED to know about it and this it is an error.